sympy/matrices/tests/test_matrices.py

```diff
@@ -2684,6 +2684,13 @@ def test_issue_9457_9467_9876():
     Q = Matrix([[1, 2, 3], [2, 3, 4], [3, 4, 5]])
     raises(IndexError, lambda: Q.col_del(-10))
 
+
+def test_issue_9480():
+    m = Matrix([[-5 + 5*sqrt(2), -5],
+                [-5*sqrt(2)/2 + 5, -5*sqrt(2)/2]])
+    assert m.rank() < m.rows
```

Comment:
If we know the rank of that specific matrix is supposed to equal 1, why not just use that for the test. The current setup would allow the test to still pass if some odd bug in the code made the rank 0 or negative. Might as well just go with the explicit `m.rank() == 1`